src/software/sensor_fusion/filter/ball_filter.h

```diff
@@ -52,25 +45,70 @@ class BallFilter
 
     /**
      * Creates a new Ball Filter
-     *
-     * @param min_buffer_size The minimum size of the buffer the filter will use to filter
-     * the ball. The buffer will shrink to this size as the ball speeds up
-     * @param min_buffer_size The maximum size of the buffer the filter will use to filter
```

Comment:
Since this comment was deleted, I think there should be a description of the buffer resizing behaviour where `MIN_BUFFER_SIZE` is defined.

Reply:
good catch.  Added comment back